src/reuse/lint.py

```diff
@@ -8,329 +8,260 @@
-                "{} of the REUSE Specification :-("
-            ).format(__REUSE_version__)
+        files_without_license = set(
+            data["non_compliant"]["missing_licensing_info"]
+        )
+        files_without_both = files_without_license.intersection(
+            files_without_copyright
         )
-    out.write("\n")
 
-    return success
+        header = (
+            "# " + _("MISSING COPYRIGHT AND LICENSING INFORMATION") + "\n\n"
+        )
+        output += header
+        if files_without_both:
+            output += _(
+                "The following files have no copyright and licensing "
+                "information:"
+            )
+            output += "\n"
+            for file in sorted(files_without_both):
+                output += f"* {file}\n"
+            output += "\n"
+
+        if files_without_copyright - files_without_both:
+            output += _("The following files have no copyright information:")
+            output += "\n"
+            for file in sorted(files_without_copyright - files_without_both):
+                output += f"* {file}\n"
+            output += "\n"
+
+        if files_without_license - files_without_both:
+            output += _("The following files have no licensing information:")
+            output += "\n"
+            for file in sorted(files_without_license - files_without_both):
+                output += f"* {file}\n"
+            output += "\n"
+
+    output += "\n"
+    output += "# " + _("SUMMARY")
+    output += "\n\n"
+
+    summary_contents = [
+        (_("Bad licenses:"), ", ".join(data["non_compliant"]["bad_licenses"])),
+        (
+            _("Deprecated licenses:"),
+            ", ".join(data["non_compliant"]["deprecated_licenses"]),
+        ),
+        (
+            _("Licenses without file extension:"),
+            ", ".join(data["non_compliant"]["licenses_without_extension"]),
+        ),
+        (
+            _("Missing licenses:"),
+            ", ".join(data["non_compliant"]["missing_licenses"]),
+        ),
+        (
+            _("Unused licenses:"),
+            ", ".join(data["non_compliant"]["unused_licenses"]),
+        ),
+        (_("Used licenses:"), ", ".join(data["summary"]["used_licenses"])),
+        (
+            _("Read errors: {count}").format(
+                count=len(data["non_compliant"]["read_errors"])
+            ),
+            "empty",
+        ),
+        (
+            _("files with copyright information: {count} / {total}").format(
+                count=data["summary"]["files_with_copyright_info"],
+                total=data["summary"]["files_total"],
+            ),
+            "empty",
+        ),
+        (
+            _("files with license information: {count} / {total}").format(
+                count=data["summary"]["files_with_licensing_info"],
+                total=data["summary"]["files_total"],
+            ),
+            "empty",
+        ),
+    ]
+
+    for key, value in summary_contents:
```

Comment:
`summary_contents` should be a dictionary if it has keys and values.

Reply:
This is fixed in the newest commit.